bin-src/lib/getDependentStoryFiles.ts

```diff
@@ -93,17 +93,28 @@ export async function getDependentStoryFiles(
   const namesById: Record<number, string> = {};
   const reasonsById: Record<number, string[]> = {};
   const csfGlobsByName: Record<string, true> = {};
-
+  const CSF_REGEX = /.\/src.*?sync/g;
+  const URL_PARAM_REGEX = /(\?.*)|(#.*)/g;
   stats.modules
     .filter((mod) => isUserModule(mod))
     .forEach((mod) => {
-      const normalizedName = normalize(mod.name);
+      let { name } = mod;
+      if (URL_PARAM_REGEX.test(mod.name) && !CSF_REGEX.test(mod.name)) {
```

Comment:
We need this `CSF_REGEX` test because the stats file generates a very weird blob for CSF files which seemed to have this "./src ... sync"and the  ? and # characters, which cause the file glob to also match the URL parameter test.